nomenclature/definitions/variable/economy/economy.yaml

```diff
@@ -4,6 +4,66 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Households|<Product>|Imported:
+   description: Consumption of imported <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Imported:
+   description: Consumption of imported <this product> by government
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by government
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<Product>|Imported:
+   description: Consumption of imported <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|GrossCapitalFormation|<Product>|Domestic:
+   description: Consumption of domestic <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Imported:
+   description: Consumption of imported <this product> by industries
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Domestic:
+   description: Consumption of domestic <this product> by industries
+   unit: million euro/yr
+
+IndustryOutput|<Industry>:
+   description: Total output on <this industry> level
+   unit: million euro/yr
+
+ProductOutput|<Product>:
+   description: Total output on <this product> level
```

Comment:
Is the output really happening *on* a *level*?

Reply:
@hettieboonman, could you bring us an additional comment to extend the description of the `ProductOutput|<Product>` variable, please?

Reply:
**Is the output really happening on a level?**
We could rewrite the descriptions as follows. Let me know if this is 

Output|Industry|<Industry>:
   description: Total output per <this industry>. That is, total value of production in <this industry>. 
   unit: million euro/yr

Output|Product|<Product>:
   description: Total output per <this product>. That is, total value of <this product> sold.

Reply:
We already have this on [here](https://github.com/openENTRANCE/nomenclature/pull/68/files):

```
Output|Industry|<Industry>:
   description: Total output on <this industry> level
   unit: million euro/yr

Output|Product|<Product>:
   description: Total output on <this product> level
```

I'll just change the description.